src/commands/init.ts

```diff
@@ -174,17 +181,17 @@ initCommand.action(async () => {
   }
 });
 
-const getClaudeConfigPath = () => {
+const getClaudeConfigFolderPath = () => {
```

Comment:
`Dir` is more common in the Node.js/JS world

```suggestion
const getClaudeConfigDirPath = () => {
```